<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputSelectorWidgetClass</class>
 <widget class="QWidget" name="InputSelectorWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InputSelectorWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1">
    <widget class="QPushButton" name="okPushButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Ok</string>
     </property>
     <property name="icon">
      <iconset resource="../../resources/main.qrc">
       <normaloff>:/new/prefix1/accept.png</normaloff>:/new/prefix1/accept.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="inputLineLabel">
     <property name="text">
      <string>InputLine:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QComboBox" name="presetsComboBox">
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>v4l2:/dev/video0,/dev/video1</string>
      </property>
      <property name="icon">
       <iconset resource="../../resources/main.qrc">
        <normaloff>:/new/prefix1/webcam.png</normaloff>:/new/prefix1/webcam.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>v4l2:/dev/video1,/dev/video0</string>
      </property>
      <property name="icon">
       <iconset resource="../../resources/main.qrc">
        <normaloff>:/new/prefix1/webcam.png</normaloff>:/new/prefix1/webcam.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>v4l2:/dev/video0,/dev/video1:1/30:fjpeg:640x480</string>
      </property>
      <property name="icon">
       <iconset resource="../../resources/main.qrc">
        <normaloff>:/new/prefix1/webcam.png</normaloff>:/new/prefix1/webcam.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>prec:data/pair/image0001%0.0s_c%d.pgm</string>
      </property>
      <property name="icon">
       <iconset resource="../../resources/main.qrc">
        <normaloff>:/new/prefix1/remove.png</normaloff>:/new/prefix1/remove.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>v4l2d:/dev/video1:1/30:rc</string>
      </property>
      <property name="icon">
       <iconset resource="../../resources/main.qrc">
        <normaloff>:/new/prefix1/3d_glasses_rc.png</normaloff>:/new/prefix1/3d_glasses_rc.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>prec:data/pair/image%04d_c%d.pgm,100ms</string>
      </property>
      <property name="icon">
       <iconset resource="../../resources/main.qrc">
        <normaloff>:/new/prefix1/remove.png</normaloff>:/new/prefix1/remove.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>dshow:0,1:1/30:fjpeg:960x544</string>
      </property>
      <property name="icon">
       <iconset resource="../../resources/main.qrc">
        <normaloff>:/new/prefix1/webcam.png</normaloff>:/new/prefix1/webcam.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>dshow:1:1/30:rc</string>
      </property>
      <property name="icon">
       <iconset resource="../../resources/main.qrc">
        <normaloff>:/new/prefix1/3d_glasses_rc.png</normaloff>:/new/prefix1/3d_glasses_rc.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ueye:1001,1002:85mhz:10b:x2:esync:global</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLineEdit" name="inputLineEdit"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>V4L2</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="6" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="v4l2StereoCheckBox">
         <property name="text">
          <string>Stereo</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QComboBox" name="v4l2WidthComboBox">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="v4l2hwCheckBox">
         <property name="text">
          <string>HxW</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="v4l2Device1ComboBox"/>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="v4l2Device1LineEdit"/>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="v4l2CodecComboBox">
         <item>
          <property name="text">
           <string>Default</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MJPEG</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Fast MJPEG</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>YUYV</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QComboBox" name="v4l2Device2ComboBox"/>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="v4l2HightComboBox">
         <property name="editable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="v4l2Device2LineEdit"/>
       </item>
       <item row="0" column="4">
        <widget class="QPushButton" name="v4l2RefreshPushButton">
         <property name="toolTip">
          <string>Refresh</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../resources/main.qrc">
           <normaloff>:/new/prefix1/refresh.png</normaloff>:/new/prefix1/refresh.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QCheckBox" name="v4l2FpsCheckBox">
         <property name="text">
          <string>FPS</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="v4l2NumComboBox">
         <item>
          <property name="text">
           <string>1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>2</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QComboBox" name="v4l2DenumComboBox">
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>15</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>20</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>25</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>60</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>50</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>AVI</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLineEdit" name="aviLineEdit"/>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="browseAviButton">
         <property name="toolTip">
          <string>Browse</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../resources/main.qrc">
           <normaloff>:/new/prefix1/folder-grey-open-24x24.png</normaloff>:/new/prefix1/folder-grey-open-24x24.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <attribute name="title">
       <string>uEye</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>File</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QLineEdit" name="fileLineEdit"/>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="browseFileButton">
         <property name="toolTip">
          <string>Browse</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../resources/main.qrc">
           <normaloff>:/new/prefix1/folder-grey-open-24x24.png</normaloff>:/new/prefix1/folder-grey-open-24x24.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../resources/main.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>okPushButton</sender>
   <signal>released()</signal>
   <receiver>InputSelectorWidgetClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>427</y>
    </hint>
    <hint type="destinationlabel">
     <x>498</x>
     <y>412</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>presetsComboBox</sender>
   <signal>activated(QString)</signal>
   <receiver>inputLineEdit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>v4l2StereoCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>v4l2Device2LineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>v4l2StereoCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>v4l2Device2ComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>v4l2Device1ComboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>v4l2Device1LineEdit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>440</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>v4l2Device2ComboBox</sender>
   <signal>currentTextChanged(QString)</signal>
   <receiver>v4l2Device2LineEdit</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>437</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>336</x>
     <y>102</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
